### PR TITLE
webapp: fixing zip deploy command to support connection_verify

### LINKED ISSUE #10216: [webapp] - Apply should_disable_connection_verify consistently
`verify=not should_disable_connection_verify()` is missing on this line:

https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/appservice/custom.py#L2330

See this line for the correct implementation:

https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/appservice/custom.py#L325

A fix was made previously, but this was only a partial solution, since the next call to check the status of the deployment fails:

https://github.com/panchagnula/azure-cli/commit/a48061ea65d71d007b6cf45cd8586047b69fb730

Hope this can be fixed soon. Thanks!

### LINKED ISSUE #10248: az webapp config set --generic-configurations failed
## Describe the bug
When adding in --generic-configuraitons "{"clientAffinityEnabled": false}" to the webapp config set commands I receive an unexpected error.

**Command Name**
--generic-configurations "{"clientAffinityEnabled": false}"

**Errors:**
```
not enough values to unpack (expected 2, got 1)
Traceback (most recent call last):
azure/cli/core/util.py, ln 244, in shell_safe_json_parse
    return json.loads(json_or_dict_string)
lib/python3.6/json/__init__.py, ln 354, in loads
    return _default_decoder.decode(s)
lib/python3.6/json/decoder.py, ln 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
lib/python3.6/json/decoder.py, ln 355, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 1 column 2 (char 1)

...

cli/command_modules/appservice/custom.py, ln 788, in update_site_configs
    config_name, value = s.split('=', 1)
ValueError: not enough values to unpack (expected 2, got 1)
```

## To Reproduce:
add --generic-configurations into the webapp config set command

- _Put any pre-requisite steps here..._
- `az webapp config set -g {} -n {} --generic-configurations {"clientAffinityEnabled": false} --slot {stagging} --ftps-state {Disabled} --php-version {5.6} --http20-enabled {true}`

## Expected Behavior
Update the ARR Affinity setting to false

## Environment Summary
```
Linux-4.9.184-linuxkit-x86_64-with
Python 3.6.4
Shell: bash

azure-cli 2.0.71

Extensions:
interactive 0.4.3

```

### PR DESCRIPTION
**Pull request by panchagnula**
fixes #10216, fixes #10248 
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by panchagnula** [comment]
@yugangw-msft can i get a PR on this to merge to the current sprint? thank you.

File: src/azure-cli/azure/cli/command_modules/appservice/_help.py [hunk 1]
```diff
@@ -861,7 +861,7 @@
         az webapp config set -g MyResourceGroup -n MyUniqueApp --always-on true
   - name: turn on "alwaysOn" through a json with content '{"alwaysOn", true}'
     text: >
-        az webapp config set -g MyResourceGroup -n MyUniqueApp --generic-configurations "{"alwaysOn": true}"
+        az webapp config set -g MyResourceGroup -n MyUniqueApp --generic-configurations '{"alwaysOn": true}'
```
Comment (yugangw-msft) [inline src/azure-cli/azure/cli/command_modules/appservice/_help.py]: this only works for bash, maybe we can do `"{\"alwaysOn\": true}"`, also you need to update line 862 as well

Comment (panchagnula) [reply src/azure-cli/azure/cli/command_modules/appservice/_help.py]: yes will do

**Comment by panchagnula** [comment]
@yugangw-msft tests unrelated to my change are failing - i am going to restart the build but not sure if this is something i can fix in this PR.

File: src/azure-cli/azure/cli/command_modules/appservice/_help.py (hunk 1 above)
Comment (qwordy) [reply src/azure-cli/azure/cli/command_modules/appservice/_help.py]: > 
> 
> yes will do

Nice to meet you

**Pull request merged**